src/test/java/com/hocs/test/pages/workstacks/Workstacks.java

```diff
@@ -139,7 +139,7 @@ public void selectSummaryTab() {
 
     public void clickCheckboxRelevantToCaseReference() {
         String caseReference =
-                sessionVariableCalled("caseReference").toString();
+                sessionVariableCalled("caseId").toString();
```

Comment:
Is `caseID` what we're going with now? I think `caseReference` is more fitting

Reply:
I've been moving all to caseId to fit methods I have done but happy to revert in a future iteration.